clients/fides-js/rollup.config.mjs

```diff
@@ -169,12 +172,21 @@ SCRIPTS.forEach(({ name, gzipErrorSizeKb, gzipWarnSizeKb, isExtension }) => {
       {
         // Intended for browser <script> tag - defines `Fides` global. Also supports UMD loaders.
         file: `dist/${name}.js`,
-        name: isExtension ? undefined : "Fides",
+        name: isExtension ? undefined : GLOBAL_NAME,
         format: isExtension ? "es" : "umd",
         sourcemap: IS_DEV ? "inline" : false,
         amd: {
           define: undefined, // prevent the bundle from registering itself as an AMD module, even if an AMD loader (like RequireJS) is present on the page. This allows FidesJS to use Rollup's `umd` format to support both `iife` and `cjs` modules, but excludes AMD.
         },
+        banner: isExtension
+          ? undefined
+          : `(function(){if(typeof ${GLOBAL_NAME}==="undefined" || (${GLOBAL_NAME}.options && ${GLOBAL_NAME}.options.fidesUnsupportedRepeatedScriptLoading === "enabled_acknowledge_not_supported")) {console.log('worked once');`,
+        footer: isExtension
```

Comment:
I'd feel _much_ more comfortable if we only added a "banner" instead of using banner/footer to wrap the entire script in an if statement. This seems to work, but it feels really fragile.

A "banner" version of this would be like
```
// FidesJS
// Before loading, check if this script has already loaded
if (...) {
  console.error(...)
  return;
}
```

Reply:
Throw an error?

Reply:
Since we're doing some pretty clever bundling, I think I prefer the DX of a thrown exception here, instead of a graceful if/else. I don't trust the banner+footer to not surprise us in so d very odd way months from now

Reply:
makes sense. I'll test that